### PR TITLE
Add Security Advice for Creating a Wallet

### PR DESCRIPTION
**Pull request by jackleeio**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING#create-a-pull-request).
- [x] A relating issue is created: #684  (<- fill the issue number here)
- [x] **This is NOT a PR to submit translation**

- - -
This change is [Reviewable](https://reviewable.io/reviews/Conflux-Chain/conflux-documentation/695)

### CODE REVIEW

File: docs/general/blockchain-security/create-wallet.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+displayed_sidebar: generalSidebar
+---
+# Security Advice for Creating a Wallet
+
+The first step into the blockchain world is creating a secure wallet. This guide provides crucial security advice for creating your wallet.
+
+## 1. Understanding the Essence of Wallets
+
+A blockchain wallet is essentially a tool for managing your private keys. The private key is a randomly generated string of numbers, while the mnemonic phrase is a human-readable version of the private key. For example:
```
Comment (darwintree) [inline docs/general/blockchain-security/create-wallet.md:10]: This is not accurate. Mnemonic is human-readable version of random seed and private key can be derived from it

File: docs/general/blockchain-security/create-wallet.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+- Private key: A seemingly random string of characters
+- Mnemonic phrase: Usually 12-24 easy-to-remember words
+
+Remember: Whoever holds the private key controls the assets in the wallet.
```
Comment (darwintree) [inline docs/general/blockchain-security/create-wallet.md:15]:
```suggestion
Remember: Whoever holds the private key controls the assets in the wallet, so never tell it to anyone else or expose it on Internet.
```

File: docs/general/blockchain-security/create-wallet.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## 2. Choosing the Right Wallet Type for Your Needs
+
+There are various wallet types, including:
```
Comment (darwintree) [inline docs/general/blockchain-security/create-wallet.md:19]: I suppose more introduction or relating articles should be added to this section. I would suppose the reader be a beginner of blockchain world and current version would make it hard for reader to understand

File: docs/general/blockchain-security/create-wallet.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+## 4. Security Measures When Creating a Wallet
+
+- Choose a secure physical environment, avoid being monitored or overlooked
+- Use a secure device, preferably new or dedicated
+- Consider creating important wallets in an offline environment
+- Verify the randomness of the generated mnemonic phrase (avoid patterns in word combinations)
+
+## 5. Adopt a Multi-Wallet Strategy to Spread Risk
+
```
Comment (darwintree) [inline docs/general/blockchain-security/create-wallet.md:53]: Add details illustrating why this strategy is required

File: docs/general/blockchain-security/create-wallet.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+- Allocate different wallets based on purpose: trading, storage, experimentation, etc.
+- Don't concentrate all assets in a single wallet
+- Use a low-value wallet to test new projects or unfamiliar DApps
+
+## 6. Understanding Emerging Keyless Wallet Technology
```
Comment (darwintree) [inline docs/general/blockchain-security/create-wallet.md:58]: More details are needed here

File: docs/espace/tutorials/oracle/Witnet/_category_.json [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (darwintree) [inline docs/espace/tutorials/oracle/Witnet/_category_.json:1]: should be removed

File: docs/espace/tutorials/oracle/Witnet/randomNumbers.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (darwintree) [inline docs/espace/tutorials/oracle/Witnet/randomNumbers.md:1]: should be removed

**Review by darwintree — CHANGES_REQUESTED** [review]
I think the general idea content is good. But current version is not good enough for blockchain beginners as too many details are not presented in this article although they seem to be common knowledge.

I suggest adding more details to the article

**Comment by jackleeio** [comment]
Updated based on feedback

File: docs/general/blockchain-security/create-wallet.md (hunk 3 above)
Comment (darwintree) [reply docs/general/blockchain-security/create-wallet.md:47]: Should add "what this is" as well as "what it is used for"

**Review by darwintree — CHANGES_REQUESTED** [review]

File: docs/general/blockchain-security/create-wallet.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+
+1. It stores your digital assets (cryptocurrencies)
+2. It allows you to send and receive these assets
+3. Most importantly, it manages your access to these assets through cryptographic keys
+
+### Key Components of a Wallet
+
+A wallet doesn't actually store your cryptocurrencies. Instead, it stores the keys that prove your ownership of these assets on the blockchain. Here are the key components:
+
+1. **Random Seed**: This is a large, randomly generated number that serves as the foundation of your wallet. It's the most crucial piece of information.
```
Comment (darwintree) [inline docs/general/blockchain-security/create-wallet.md:24]: What does this mean?

**Review by darwintree — CHANGES_REQUESTED** [review]

**Review by darwintree — APPROVED** [review]

**Pull request merged**